src/containers/SidebarControl/SidebarControl.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+
+import { AppContext } from "./../../App";
+import { Box, IconButton, Grid, Tooltip } from "@material-ui/core";
+import { iconStyle } from "./../../styles/global";
+import ChevronLeftIcon from "@material-ui/icons/ChevronLeft";
```

Comment:
you can destructure icons similarly how you do material-ui components and import them something like
import { ChevronLeft as ChevronLeftIcon, ChevronRight as ChrevronRightIcon } from "@material-ui/icons";

Reply:
This does not seem to work for me. Icons are not being exported from "@material-ui/icons".